cmd/_integration-tests/transport/transport-test.proto

```diff
@@ -27,8 +27,15 @@ service TransportPermutations {
       body: "*"
     };
   }
+  rpc ErrorRPC (Empty) returns (Empty) {
+    option (google.api.http) = {
+      get: "/error"
+    };
+  }
 }
 
+message Empty {}
```

Comment:
In the current version of truss it is not possible to import that file. As when truss invokes `protoc` it does not import the `GOPATH` and such cannot find that file.

Though when pull #92 gets merged in, this file could be imported. Though the import line will be longer than the line that redefines `Empty`.